src/cli/main.cpp

```diff
@@ -213,11 +213,11 @@ int main(int argc, char* argv[]) {
       rubyInterpreter.evalString(embedded_extensions_string);
     } catch (const std::exception& e) {
       rubyInterpreter.evalString(R"(STDOUT.flush)");
-      std::cout << "Exception in embedded_help: " << e.what() << std::endl;  // endl will flush
+      std::cout << "Exception in embedded_help: " << e.what() << '\n';  // endl will flush
```

Comment:
Might want some of these to flush, since this would result in ending the program you'd want the output written before exit

Reply:
:+1: thank you